### ISSUE TITLE
Hidden Service down/unreachable

### ISSUE THREAD

**Issue by willcl-ark** [bitcoin-core/bitcoincore.org#985]
As reported by @TNTBOMBOM in https://github.com/bitcoin/bitcoin/issues/28054 the bitcoincore.org hidden service http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/ is currently unreachable using Tor Browser.

I have verified that it is unreachable, and many other Tor websites remain accessible, indicating that it is not a problem with the Tor network itself.

**Comment by katesalazar** [comment]
repro

**Comment by fanquake** [comment]
@willcl-ark Is this still the case?

**Comment by willcl-ark** [comment]
@fanquake it is still the case for me.

You get the nice "onion available" button if you visit the http site using Tor browser, but it never loads that way either (of course, it's the same address)

![image](https://github.com/bitcoin-core/bitcoincore.org/assets/6606587/48447fd2-ac4b-4cbb-a352-dbefd2512d24)

```
The operation timed out when attempting to contact 6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion.
```

**Comment by katesalazar** [comment]
still repro

I can reach ddg hidden service example, but not bitcoincores one

**Comment by BenWestgate** [comment]
Still down.

**Comment by TNTBOMBOM** [comment]
Meow?!

**Comment by f321x** [comment]
still down here :(

**Comment by Sjors** [comment]
It's still down. If it can't be fixed, then we should remove the suggested Tor browser redirect:

https://github.com/bitcoin-core/bitcoincore.org/blob/f1d9b75ccd6e0a7762175e76956d6f9aff18dc1a/_includes/head.html#L14

**Comment by willcl-ark** [comment]
Yeah, it seems a (slight) shame that it seems to be difficult to fix, but broken is worse than nothing?

It is also possible to access the clearnet site via an exit node. Although this *feels* worse, in reality users should be verifying their downloads using GPG not trusting any particular web addresses. If they are doing gpg verification then the impact of dropping the hidden service is pretty minimal, minus:

* exit node risks
  * MITM attacks (in theory mitigated by GPG)
  * other exit node attacks
* longer round-trip circuits

I did make an attempt a few weeks ago at a "simple Tor proxy" which could be deployed to a droplet. I'm assuming the current Tor site is hosted stacked with the clearnet site making maintenance difficult, and my standalone proxy therefore aims to make management/deployment (possibly) easier. A running example of that is at the bottom of *README.md*. NB that this is a **demo** site, not in any way official, and likely contains all manner of critical hidden service misconfigurations etc.

**Comment by achow101** [comment]
It should now be reachable.

**Comment by Sjors** [comment]
Indeed, works!

**Comment by willcl-ark** [comment]
Thanks @achow101 ❤
